### PR TITLE
adds doc for IronWorker Cluster API

### PR DESCRIPTION
**Pull request by derekschultz**

### CODE REVIEW

File: worker/hybrid/cluster_api.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Cluster API for IronWorker
```
Comment (arschles) [inline worker/hybrid/cluster_api.md:1]: @derekschultz can you put 1-2 sentences in here with high level context of what this API is for

**Comment by derekschultz** [comment]
Sure thing @arschles -- added description. Feedback welcome.

**Comment by arschles** [comment]
looks good to me thanks @derekschultz

**Comment by arschles** [comment]
LGTM

**Pull request merged**